lib/config/options/index.ts

```diff
@@ -101,31 +102,31 @@ const options: RenovateOptions[] = [
   {
     name: 'onboardingConfigFileName',
     description:
-      'Change this value in order to override the default onboarding config file name.',
+      'Change this value to override the default onboarding config file name.',
     type: 'string',
     default: 'renovate.json',
     globalOnly: true,
     cli: false,
   },
   {
     name: 'onboardingNoDeps',
-    description: 'Onboard repository even if no dependencies found.',
+    description: 'Onboard the repository even if no dependencies are found.',
     type: 'boolean',
     default: false,
     globalOnly: true,
   },
   {
     name: 'onboardingPrTitle',
     description:
-      'Change this value in order to override the default onboarding PR title.',
+      'Change this value to override the default onboarding PR title.',
     type: 'string',
     default: 'Configure Renovate',
     globalOnly: true,
     cli: false,
   },
   {
     name: 'productLinks',
-    description: 'Links which are embedded within PRs, issues, etc.',
+    description: 'Links which are used within PRs, issues.', // I don't think etc. applies here, are there other places where Renovate can put links???
```

Comment:
```suggestion
    description: 'Links which are used within PRs, issues.',
```

Potentially in issue and PR comments, although I'm not sure if we do that

Reply:
Potentially. I think we should either leave in `etc` or replace it with `comments`. There may also be links in status check comments